File: Aula 09/Exemplo1.py
salario = []
contador = 0
for i in range(6):
    sal = input(f"Digite o salário do {i+1}º funcionário: ")
    salario.append(sal)

for i in range(6):
    print(f"Salario do {i+1}º funcionário: R$ {salario[i]}")
    if (salario[i] > 1000):
        contador+=1

File: Aula 09/Exemplo1.java
import java.util.Scanner;

public class Exemplo1{
    public static void main(String[] args) {
        float salario[] = new float[5];
        Scanner sc = new Scanner(System.in);
        
        for(int i=0; i<5; i++){
            System.out.print("Digite o salário do " +(i+1)+"º funcionário: R$ ");
            salario[i] = sc.nextFloat();
        }
        for(int i=0; i<5; i++){
            System.out.println("o salário do do "+(i+1)+"º funcionário é: " + salario[i]);
        }        
    }
}